src/components/scenes/FioAddressListScene.js

```diff
@@ -129,6 +130,7 @@ class FioAddressList extends React.Component<Props, LocalState> {
                 <FioNameRow
                   key={`${address.name}`}
                   name={address.name}
+                  bundles={address.bundles}
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name